gc/gc-base/src/main/java/org/projectnessie/gc/expire/local/DefaultLocalExpire.java

```diff
@@ -67,10 +67,15 @@ public DeleteSummary expire() {
     Instant started = clock().instant();
     expireParameters().liveContentSet().startExpireContents(started);
 
-    ForkJoinPool forkJoinPool = new ForkJoinPool(parallelism());
+    ForkJoinPool forkJoinPool = null;
     try {
-      DeleteSummary deleteSummary =
-          forkJoinPool.invoke(ForkJoinTask.adapt(this::expireInForkJoinPool));
+      DeleteSummary deleteSummary;
+      if (parallelism() == 1) {
+        deleteSummary = this.expireInForkJoinPool();
```

Comment:
nit: the method name is weird now as it is not using the `ForkJoinPool` always. 
Can be called as `expireContents`